R/build_copy_queries.R

```diff
@@ -53,7 +65,7 @@ build_copy_queries <- function(dest, dm, set_key_constraints = TRUE, temporary =
 
       # SQL Server:
       if (is_mssql(dest)) {
-        types[pk_col] <- "INT IDENTITY"
+        types[pk_col] <- paste(types[pk_col], "IDENTITY")
```

Comment:
I had always regretted using `paste()` instead of `paste0()` .

```suggestion
        types[pk_col] <- paste0(types[pk_col], " IDENTITY")
```